### ISSUE TITLE
EPERM rename error on Windows 10 and 11 during quire build

### ISSUE THREAD

**Issue by geealbers** [thegetty/quire#626]
### Before proceeding, check to make sure there isn’t an existing issue for this bug.

- [X] I have searched the existing issues and determined this is a new bug.

### Steps to reproduce

1. Start a new Quire project on Windows
2. Run `quire build`

### Actual behavior

The following error occurs near the end of the process and the build isn't complete. The project is left with a `_site` folder with built files (but css and js haven't been bundled yet) and an empty `.11th-vite`  folder.

```sh
[11ty] Problem writing Eleventy templates: (more in DEBUG output)
[11ty] EPERM: operation not permitted, rename 'C:\Users\geeal\test-reload\_site' -> 'C:\Users\geeal\test-reload\.11ty-vite' (via Error)
[11ty]
[11ty] Original error stack trace: Error: EPERM: operation not permitted, rename 'C:\Users\geeal\test-reload\_site' -> 'C:\Users\geeal\test-reload\.11ty-vite'
```

### Expected behavior

The build should complete without errors. A `_site` directory should remain, with all the site files processed and bundled.

### Version numbers

Quire v1.0.0-rc.0
Node v18.12.1
npm v8.19.2
Windows 10 or 11

**Comment by geealbers** [comment]
In progress for v1.0.0-rc.1: https://jira.getty.edu/browse/DEV-13837

**Comment by Erin-Cecele** [comment]
Closed with PR #629

**Comment by Erin-Cecele** [comment]
Backlogged. Planned for v1.0.0-rc.2

**Comment by Erin-Cecele** [comment]
This also came up on our Discussion board (see #684) form @kanotson. Originally identified as an issue with Windows 11 but it is affecting Windows 10 as well.

**Comment by geealbers** [comment]
This has been identified as [an issue with the 11ty Vite Plugin](https://github.com/11ty/eleventy-plugin-vite/issues/22), a possible fix seems to have been found and there's hopefully a forthcoming PR to address it.

**Comment by geealbers** [comment]
Here's a temporary workaround to this issue:

**1. In the `node_modules/@11ty/eleventy-plugin-vite/EleventyVite.js` file in your project, add `await fsp.rm(tmp, { recursive: true, force: true });` between lines 51 and 52:**

<img width="815" alt="Screen Shot 2023-05-10 at 4 33 41 PM" src="https://github.com/thegetty/quire/assets/7796401/f41f330f-6afc-4de7-ae9c-748dc39e6ad6">

**2. And then on line 63 of the same file, change `path.sep` to `/`:**

<img width="806" alt="Screen Shot 2023-05-10 at 4 34 24 PM" src="https://github.com/thegetty/quire/assets/7796401/ab6e3ab5-f7f6-4d90-9e1c-c6a9d329e198">

**3. Save the changes and `quire build` should work as expected.**

**Comment by geealbers** [comment]
There are now two open PRs for this in the `eleventy-plugin-vite` repo, but no response from the maintainer yet. See https://github.com/11ty/eleventy-plugin-vite/pull/34 and https://github.com/11ty/eleventy-plugin-vite/pull/35.

**Comment by Erin-Cecele** [comment]
An additional pull request addressing this issue has been added to the `eleventy-plugin-vite` repo: https://github.com/11ty/eleventy-plugin-vite/pull/37. We are waiting for a response from the maintainer. We will revisit the issue in a few weeks for an update.

**Comment by mphstudios** [comment]
Fixed with #974 `quire-11ty@1.0.0-rc.22`
